<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pupo 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Tabs_and_QuitButton">
     <item>
      <layout class="QHBoxLayout" name="TabsLayout">
       <item>
        <widget class="QTabWidget" name="Tabs">
         <property name="toolTip">
          <string/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_del_files">
          <property name="accessibleName">
           <string/>
          </property>
          <attribute name="title">
           <string>Очистка</string>
          </attribute>
          <widget class="QPushButton" name="runBtn">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>420</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Выполнить</string>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>651</width>
             <height>131</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="del_line_edit">
                <property name="toolTip">
                 <string>Путь к директории, в которой необходимо провести операцию</string>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="placeholderText">
                 <string>Укажите директорию</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reviewBtn">
                <property name="text">
                 <string>Обзор</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="cb_del_files">
                <property name="text">
                 <string>Удалить файлы с расширением:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="extension_line_edit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>.txt</string>
                </property>
                <property name="maxLength">
                 <number>7</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>370</width>
                  <height>34</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="cb_del_empty_dir">
                <property name="text">
                 <string>Удалить пустые папки</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QTextEdit" name="output_clean">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>150</y>
             <width>651</width>
             <height>261</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_move_files">
          <attribute name="title">
           <string>Перемещение </string>
          </attribute>
          <widget class="QPushButton" name="runBtn_2">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>420</y>
             <width>71</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Выполнить</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cut_checkBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>191</width>
             <height>17</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>Удалить файлы начальной директории</string>
           </property>
           <property name="text">
            <string>Вырезать файлы</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>651</width>
             <height>91</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="handleWidth">
            <number>0</number>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="path_from_move">
               <property name="toolTip">
                <string>Путь, откуда перемещать файлы</string>
               </property>
               <property name="placeholderText">
                <string>Укажите директорию</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reviewBtnFrom">
               <property name="text">
                <string>Обзор</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_2">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="path_to_move">
               <property name="toolTip">
                <string>Путь, куда перемещать файлы</string>
               </property>
               <property name="placeholderText">
                <string>Укажите директорию</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reviewBtnTo">
               <property name="text">
                <string>Обзор</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QTextEdit" name="output_move_files">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>651</width>
             <height>281</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="concat">
          <attribute name="title">
           <string>Склейка</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>661</width>
             <height>431</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_count">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0" colspan="3">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLineEdit" name="concat_line_edit">
                  <property name="toolTip">
                   <string>Путь, где находятся файлы для склейки</string>
                  </property>
                  <property name="placeholderText">
                   <string>Укажите директорию</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="reviewBtn_concat">
                  <property name="text">
                   <string>Обзор</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="findBtn">
                  <property name="text">
                   <string>Поиск</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLineEdit" name="save_as_line_edit">
                  <property name="toolTip">
                   <string>Куда сохранить результат</string>
                  </property>
                  <property name="placeholderText">
                   <string>Укажите директорию</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="reviewBtn_save_as">
                  <property name="text">
                   <string>Обзор</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_11"/>
              </item>
              <item row="2" column="0">
               <widget class="QLineEdit" name="filename_line_edit">
                <property name="toolTip">
                 <string>Имя файла, который получится в результате склейки</string>
                </property>
                <property name="text">
                 <string>Res</string>
                </property>
                <property name="placeholderText">
                 <string>Имя файла</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="profiles_checkBox">
              <property name="toolTip">
               <string>Каждый файл в отдельный лист</string>
              </property>
              <property name="text">
               <string>Профили</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Выбрано:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QTextEdit" name="output_concat">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="list_results">
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="select_all">
                  <property name="text">
                   <string>Выделить всё</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cancel_all">
                  <property name="text">
                   <string>Отменить всё</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="run_concat">
                  <property name="text">
                   <string>Выполнить</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="QuitButtonLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 223, 223);</string>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_dir"/>
   </widget>
   <widget class="QMenu" name="about_menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_text"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="about_menu"/>
  </widget>
  <action name="about_text">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="open_dir">
   <property name="text">
    <string>Открыть </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
